src/platforms/apple/common/install/carthage.mdx

```diff
@@ -3,14 +3,13 @@ title: Carthage
 description: "Learn about integrating Sentry into your Xcode project using Carthage."
 ---
 
-If you're using Sentry via Carthage on Macs with Apple Silicon, please use Carthage >= 0.37.0, which introduced [support for XCFramework](https://github.com/Carthage/Carthage/releases/tag/0.37.0).
-
 To integrate Sentry into your Xcode project using Carthage, specify it in your _Cartfile_:
 
 ```ruby
 github "getsentry/sentry-cocoa" "{{ packages.version('sentry.cocoa') }}"
 ```
 
-Run `carthage update` to download the framework and drag the built _Sentry.framework_ into your Xcode project.
+Run `carthage update` to download the framework and drag the built _Sentry.framework_ into your Xcode project. We also provide a pre-built version for every release which can be downloaded at [releases on GitHub](https://github.com/getsentry/sentry-cocoa/releases).
 
-We also provide a pre-built version for every release which can be downloaded at [releases on GitHub](https://github.com/getsentry/sentry-cocoa/releases).
+## Macs with Apple Silicon and XCFrameworks
+If you're using Sentry via Carthage on Macs with Apple Silicon or want to use XCFrameworks, please use Carthage >= 0.37.0, which introduced [support for XCFramework](https://github.com/Carthage/Carthage/releases/tag/0.37.0). Make sure to use `carthage build --use-xcframeworks --no-use-binaries` after running `carthage update --use-xcframeworks`. Carthage 0.37.0 needs to rebuild the XCFramework form source because it can't handle Github dependencies that download binaries. For more information checkout their [release notes](https://github.com/Carthage/Carthage/releases/tag/0.37.0).
```

Comment:
```suggestion
If you're using Sentry via Carthage on Macs with Apple Silicon or want to use XCFrameworks, please use Carthage >= 0.37.0, which introduced [support for XCFramework](https://github.com/Carthage/Carthage/releases/tag/0.37.0). Make sure to use `carthage build --use-xcframeworks --no-use-binaries` after running `carthage update --use-xcframeworks`. Carthage 0.37.0 needs to rebuild the XCFramework from source because it can't handle GitHub dependencies that download binaries. Learn more in their [release notes](https://github.com/Carthage/Carthage/releases/tag/0.37.0).
```